snuba/query/parser/expressions.py

```diff
@@ -61,44 +73,94 @@ def visit_function_name(self, node: Node, visited_children: Iterable[Any]) -> st
     def visit_column_name(self, node: Node, visited_children: Iterable[Any]) -> Column:
         return Column(None, None, node.text)
 
-    def visit_empty(
+    def visit_low_pri_op(
+        self, node: Node, visited_children: Iterable[Any]
+    ) -> LowPriOperator:
+        if node.text == "+":
+            return LowPriOperator.PLUS
+        else:
+            return LowPriOperator.MINUS
```

Comment:
Actually you should check that the else condition is a `-` and raise an exception otherwise.

Reply:
Isn't it guaranteed that we would never get here unless node.text is either "+" or "-"?
If so, why not just `return LowPriOperator(node.text)` and omit the conditionals?

Reply:
Of that's actually right I forgot about the enum constructor. @ayirr7 please do what Lyn said and ignore what I said.

Reply:
Seems like a good idea!